modules/oe_whitelabel_helper/oe_whitelabel_helper.info.yml

```diff
@@ -6,8 +6,8 @@ core_version_requirement: ^9.2
 
 dependencies:
   - drupal:node
+  - drupal:twig_field_value
   - oe_bootstrap_theme:oe_bootstrap_theme_helper
-  - openeuropa:oe_corporate_blocks
```

Comment:
This was correct, why did we remove it?
We need to clean the list of modules to enable manually in the runner, not fix the dependencies here.
Only our own modules, or modules that are not supposed to mandatory, should be enabled manually (which for us is the runner.yml).